### ISSUE TITLE
GCE kube-up.sh reports error if missing python yaml

### ISSUE THREAD

**Issue by derekwaynecarr** [GoogleCloudPlatform/kubernetes#5258]
I tried to spin up GCE kube environment using kube-up and got the following error:

```shell
Traceback (most recent call last):
  File "cluster/../cluster/gce/../../cluster/gce/kube-env.py", line 19, in <module>
    import yaml
ImportError: No module named yaml
```

The script would then hang on a subsequent command.

The verify-prereqs for kube-up should require yaml support for python and fail early.

To rectify, I had to do the following:

```shell
sudo yum install python-yaml
```

**Comment by alex-mohr** [comment]
Thanks for the report Derek!

**Comment by zmerlynn** [comment]
Yeah, this broke our Jenkins a while ago, and I thought I fixed it, but didn't. The real fix is in #5273
